com.reprezen.swagedit/src/com/reprezen/swagedit/json/JsonDocumentManager.java

```diff
@@ -61,24 +65,24 @@ public JsonNode getDocument(URL url) {
         JsonNode document;
         if (url.getFile().endsWith("json")) {
             try {
-                document = mapper.readTree(url);
+                document = doRead(mapper, url);
             } catch (Exception e) {
                 document = null;
```

Comment:
Current behaviour on this task branch: 
When I open a model with non-UTF-8 symbols the symbols are silently replaced by the `�`, nothing indicates it - no entries in the error log, no markers in the problem view. In many European languages non-UTF symbols are frequent and loosing them is not what users expect.

While not showing an error dialog is an definite improvement, probably we still should report that something went wrong. 
@tedepstein we need your feedback here on how to indicate that a file has unexpected non-UTF-8 symbols and communicate why they are not allowed. I will list the most obvious options:
1. Display an error dialog on file opening. Yes, this is exactly what we removed in this pull request, but we can make the error dialog more informative and show it only once - on file opening.
2. Show warnings markers. Ideally, they should be shown on the lines where we meet invalid symbols, but I can't tell you effort estimate for this.
3. Report it in the error log. It's probably the easiest to implement, but it's also the least visible.